README.md

```diff
@@ -154,6 +152,33 @@ YACHT can be run via the command line `yacht <module>`. Now it has three four ma
   | k                 | the length of k-mer |
   | outfolder         | the path to a folder where the downloaded file is expected to locate |
 
+- The `sketch` module has two submodules: `ref` and `sample`:
+  
+  + `ref` is used to sketch fasta files and make them as a reference database
+  ```bash
+  # Example for sketching multiple fasta files as reference genomes in a given folder
+  yacht sketch ref --infile ./demo/ref_genomes --kmer 31 --scaled 1000 --outfile ref.sig.zip
+
+  ```
+  | Parameter         | Explanation                                                  |
+  | ----------------- | ------------------------------------------------------------ |
+  | infile            | the path to a input FASTQ file or a folder containing multiple FASTQ files |
+  | kmer              | the length of k-mer |
+  | scaled            | the scaled factor |
+  | outfile           | the path to a output file |
+
+  + `sample` is used to sketch the single-end or paired-end fasta file(s) and make it/them as a query sample.
+  ```bash
+  # Example for sketching a FASTA/Q file as a metagenomic example
+  yacht sample --infile ./query_data/query_data.fq --kmer 31 --scaled 1000 --outfile sample.sig.zip
```

Comment:
This should be `yacht sketch sample`, right?

Reply:
Thanks for catching this typo. I have fixed it.